main.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-use-before-define */
+// eslint-disable-next-line max-classes-per-file
+const listOfBooks = document.querySelector('.book-list');
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.bookid = Math.random().toFixed(1);
+  }
+}
+
+class StoreBook {
+  constructor() {
+    // Array of book items
+    this.BookData = [];
+  }
+
+  // New books
+  addBook = (newbook) => {
+    this.BookData.push(newbook);
+    localStorage.setItem('AllBooks', JSON.stringify(this.BookData));
+    DisplayBooks(newbook);
+  };
+
+  // Remove book from the list
+  removeBook = (bookid) => {
+    const deleteBook = document.getElementById(bookid);
+    deleteBook.remove();
+    this.BookData = this.BookData.filter((x) => x.bookid !== bookid);
+    localStorage.setItem('AllBooks', JSON.stringify(this.BookData));
+  };
+}
+
+const savebook = new StoreBook();
+// Get input value
+const getformInput = () => {
+  const title = document.querySelector('.title');
+  const author = document.querySelector('.author');
+  const insertbook = new Book(title.value, author.value);
+  return insertbook;
+};
```

Comment:
With this function the way it is, it's possible to add a new book with an empty author and title. I suggest you check if the inputs are empty before moving ahead with creating the new book. Something like this should do the trick: 
```
if (title.value !== "" && author.value !== "") {
    const insertbook = new Book(title.value, author.value);
    return insertbook;
}
```

Reply:
Modified the code as suggested to preventing submitting empty author and title.

Reply:
## 👍